test/unit/date-convert.test.js

```diff
@@ -10,4 +19,9 @@ describe('convert currency', () => {
     const result = convertToDaxDate('2021-11-01')
     expect(result).toEqual('01/11/2021')
   })
+
+  test('uses current date in correct format if no date supplied', () => {
```

Comment:
Could you add other tests to cover other scenarios and data types to give us confidence of defence in depth for future integrations:

- '2021/11/01'
- '01-11-2021'
- '01-11-2021'
- 'Monday 1st November 2021'
- '32/11/2021'
- undefined
- null
- true
- false
- {}
- []
- 1
- 0
- Boolean()

Reply:
Tests added, I've also updated to handle different date formats.  The allowed date formats have also been updated in `asyncapi` spec.